spec/models/spree/payment/processing_decorator_spec.rb

```diff
@@ -14,5 +14,12 @@
 
       expect(payment.close!).to be_truthy
     end
+
+    it 'payment receives close! method' do
+      payment = create(:amazon_payment, state: 'completed')
+
+      expect(payment).to receive(:close!)
```

Comment:
When I stub the close request I get:

`
1) Spree::Payment::Processing#close! delegates close! to the payment source
     Failure/Error: expect(payment.source).to have_received(:close!)
       #<Spree::AmazonTransaction id: 1, order_id: nil, order_reference: "ORDER_REFERENCE", authorization_id: "AUTHORIZATION_ID", capture_id: "CAPTURE_ID", closed_at: "2016-09-27 18:57:27"> expected to have received close!, but that object is not a spy or method has not been stubbed.
     # ./spec/models/spree/payment/processing_decorator_spec.rb:35:in `block (3 levels) in <top (required)>'
`